pkg/machines/register.go

```diff
@@ -58,7 +60,7 @@ func CustomResourceDefinitionExists(name string, clientset apiextensionsclient.I
 }
 
 func createCustomResourceDefinition(plural, kind string, clientset apiextensionsclient.Interface) error {
-	name := plural + "." + v1alpha1.GroupName
+	name := CRDName
```

Comment:
Is that really necessary?

Reply:
The reason was I wanted to export it. I can also revert this part back and just hardcode the GVK in the migration/conversion

Reply:
I'm fine with the const - definitely a improvement. But do we still need to local `name` variable then?